jenkins/ci-dnf-stack.yaml

```diff
@@ -0,0 +1,132 @@
+- project:
```

Comment:
Please can you explain, how to use it in readme.md file for others who would like to use our project, or for future work. Short description with external link also should work.